<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserGUI</class>
 <widget class="QWidget" name="UserGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UserGUI</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="nameLabel">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Name</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nameLine"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="genreLabel">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Genre</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="genreLine"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="yearLabel">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Year</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="yearLine"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="likesLabel">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>11</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Likes</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="likesLine"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="trailerLabel">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Trailer</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="trailerLine"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="filterLabel">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Genre filter</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="filterLine"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="watchlist">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="2">
        <widget class="QPushButton" name="removeButton">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="addButton">
         <property name="text">
          <string>Add to watchlist</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="nextButton">
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(94, 211, 65, 255), stop:1 rgba(255, 255, 255, 255));</string>
         </property>
         <property name="text">
          <string>Next</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="filterButton">
         <property name="text">
          <string>List by genre</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="openButton">
         <property name="text">
          <string>Open in file</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="likeButton">
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(144, 153, 211, 255), stop:1 rgba(255, 255, 255, 255));</string>
         </property>
         <property name="text">
          <string>Like</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
